pkgs/applications/video/obs-studio/obs-gstreamer.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gstreamer
+, gst-plugins-base
+, pkgconfig
+, meson
+, ninja
+, obs-studio
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-gstreamer";
+  version = "v0.1.0";
```

no v in version :wink: 
```suggestion
  version = "0.1.0";
```

---

thank you, committed.